nextflow_schema.json

```diff
@@ -38,6 +38,118 @@
                 }
             }
         },
+        "database": {
+            "title": "Databases",
+            "type": "object",
+            "description": "Select databases to be run on all samples.",
+            "fa_icon": "fas fa-terminal",
+            "properties": {
+                "pointfinder_database": {
+                    "enum": [
+                        "Automatic Selection",
+                        "Enterococcus faecium",
+                        "Enterococcus faecalis",
+                        "Helicobacter pylori",
+                        "Salmonella",
+                        "Campylobacter",
+                        "Escherichia coli"
```

Comment:
I think it's okay if these don't exactly match the command line expectations (ex: `enterococcus_faecium`), because you're converting all these into those strings, right?

Reply:
Yes, your original `convert` function in `conf/modules.cong` tackled [this](https://github.com/phac-nml/staramrnf/blob/8bc52312ca8f5e7526ee5f628e902c6699ae62c6/conf/modules.config#L47), and I have since made it a bit more broad to accept metadata output by Kraken2 for species.